metacat-metadata-mysql/src/main/java/com/netflix/metacat/metadata/mysql/MySqlServiceUtil.java

```diff
@@ -51,18 +43,16 @@ private MySqlServiceUtil() {
     /**
      * Returns the list of string having the input ids.
      *
-     * @param jdbcUtil jdbc util
+     * @param jdbcTemplate jdbc template
      * @param sql      query sql
      * @param item     identifier
      * @return list of results
-     * @throws DataAccessException data access exception
      */
-    @Transactional(readOnly = true)
-    public static Set<String> getValues(final JdbcUtil jdbcUtil,
+    public static Set<String> getValues(final JdbcTemplate jdbcTemplate,
```

Comment:
It looks that Spring declarative transaction [does not support static method](https://stackoverflow.com/questions/23569732/transactional-with-static-method). I am not sure if this utility class will work here. You might want to test it out.

Reply:
Are we still using this one?